fabtests/benchmarks/benchmark_shared.c

```diff
@@ -333,7 +333,7 @@ int pingpong_rma(enum ft_rma_opcodes rma_op, struct fi_rma_iov *remote)
 			if (rma_op == FT_RMA_WRITE)
 				*(tx_buf + opts.transfer_size - 1) = (char)i;
 
-			if (opts.transfer_size <= inject_size)
+			if (opts.transfer_size <= inject_size && opts.transfer_size)
```

Comment:
The commit message needs better phrase... I think the point is there is meaning to test 0 byte with inject as it will have no completion at all. Doing 0 byte w/o inject is more meaningful as it can be used as a sync

Reply:
Also the `opts.transfer_size` should be at the beginning of the `if`